README.md

```diff
@@ -32,16 +32,16 @@ You are now ready to start the server, run
 There are 3 major segments of the applications API
 
 1. The Authentication / Authorization controller - to handle functions such as login / register / logout / token rotation
-2. The user controller - to handle retrieval of user information
-3. The wallet controller - to handle wallet topup and transfer of funds to other users
+2. The user controller - to handle the retrieval of user information
+3. The wallet controller - to handle wallet top-up and transfer of funds to other users
 
 
--  ### **Lets get started with Auth**
+-  ### **Let's get started with Auth**
```

Comment:
**Suggestion:** Add a space after the bullet point in line 39 for better readability.
```suggestion
- ### **Let's get started with Auth**
```